purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Tecnativa - Manuel Calero
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo.tests.common import TransactionCase
+from odoo import fields
+
+
+class TestPurchaseOrderUninvoiceAmount(TransactionCase):
```

Comment:
use SavePointCase for better test performance

Reply:
Done